app/models/upload_saver.rb

```diff
@@ -0,0 +1,33 @@
+require "fileutils"
+
+# Saves uploaded files.
+class UploadSaver
+  STORAGE_PATH = Rails.root.join(*%w(tmp uploads))
```

Comment:
Could we change this to `Rails.root.join("tmp", "uploads")`? It's actually the same number of characters and is a little less mechanically complex.